src/Main.java

```diff
@@ -1,5 +1,63 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("task 1");
+        for (int i = 0; i <= 10; i++) {
```

Comment:
```suggestion
        for (int i = 1; i <= 10; i++) {
```
от 1 до 10 по заданию требовалось